ihmc-high-level-behaviors/src/main/java/us/ihmc/behaviors/activeMapping/StancePoseCalculator.java

```diff
@@ -87,10 +74,10 @@ public void insertCandidatePoses(ArrayList<FramePose3D> poses, FramePose3D goalP
       }
    }
 
-   public void searchForOptimalGoalStance(ArrayList<FramePose3D> leftPoses, ArrayList<FramePose3D> rightPoses, FramePose3D goalPose, TerrainMapData terrainMap)
+   public void searchForOptimalGoalStance(ArrayList<FramePose3D> leftPoses, ArrayList<FramePose3D> rightPoses, FramePose3D pickPoint, TerrainMapData terrainMap)
```

Comment:
You can make pickpoint (one: a new name), and then make it `FramePose3DReadOnly`